src/main/java/com/exasol/cloudwatch/ExasolStatisticsTableMetricReader.java

```diff
@@ -0,0 +1,145 @@
+package com.exasol.cloudwatch;
+
+import java.sql.*;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.exasol.errorreporting.ExaError;
+
+/**
+ * This class reads the value of {@link ExasolStatisticsTableMetric}s from the Exasol database.
+ */
+public class ExasolStatisticsTableMetricReader {
+    private static final TimeZone UTC_ZONE = TimeZone.getTimeZone("UTC");
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExasolStatisticsTableMetricReader.class);
+    private final Connection connection;
+    private final String schemaOverride;
+    private final Calendar utcCalendar;
+
+    /**
+     * Create a new instance of {@link ExasolStatisticsTableMetricReader}.
+     * 
+     * @param connection     connection to the exasol database
+     * @param schemaOverride if null EXA_STATISITCS is used. This parameter allows you to test this connector with a
+     *                       predefined SCHEMA instead of the unmodifiable live statistics.
+     */
+    public ExasolStatisticsTableMetricReader(final Connection connection, final String schemaOverride) {
+        this.connection = connection;
+        this.schemaOverride = schemaOverride;
+        this.utcCalendar = Calendar.getInstance(UTC_ZONE);
+        setSessionTimezoneToUtc();
+    }
+
+    private void setSessionTimezoneToUtc() {
```

Comment:
Do you still need this? I see that in the queries you already use convert_tz function.

Reply:
yes. It's relevant for the output of NOW(); That returns a value of the type `TIMESTAMP WITH LOCAL TIMEZONE`. For all other queries, you're right. There it does not change anything.